android/app/src/main/java/com/mattermost/rnbeta/MainApplication.java

```diff
@@ -34,7 +35,8 @@
 
 public class MainApplication extends NavigationApplication implements INotificationsApplication {
 
-  NotificationsLifecycleFacade notificationsLifecycleFacade;
+  public static MainApplication instance;
```

Comment:
Not clear on the use of this static variable. It looks like you're trying to make `MainApplication` a singleton, but you aren't actually using it anywhere. You assign `this` to `instance` on line 70, but you don't use it on line 60, instead using `MainApplication.this`. 

I think you can probably remove `instance` altogether and just pass `this` on line 60.